doc/src/examples/getting_started/CMakeLists.txt

```diff
@@ -6,6 +8,23 @@ find_package(CppMicroServices REQUIRED)
 
 include(GenerateExportHeader)
 
+# Detect "local" build (build from getting started directory)
+if (CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+  set(IS_LOCAL_BUILD ON)
+endif()
+
+if (IS_LOCAL_BUILD)
+  # Set binary and library output directories
+  set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/lib")
+  set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/lib")
+  set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
+
+  set(BUILD_SHARED_LIBS ON CACHE BOOL "Build shared libraries")
+  set(US_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
+
+  set(CMAKE_ENABLE_EXPORTS ON)
+endif()
```

Comment:
is the `IS_LOCAL_BUILD` variable needed? can't it be one if statement?

```
if (CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
  # Set binary and library output directories
  set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/lib")
  set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/lib")
  set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")

  set(BUILD_SHARED_LIBS ON CACHE BOOL "Build shared libraries")
  set(US_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})

  set(CMAKE_ENABLE_EXPORTS ON)
endif()
```

Reply:
It’s not.